src/app/clusters/refrigerator-alarm-server/refrigerator-alarm-server.h

```diff
@@ -40,6 +42,10 @@ class RefrigeratorAlarmServer
     EmberAfStatus SetStateValue(chip::EndpointId endpoint,
                                 chip::BitMask<chip::app::Clusters::RefrigeratorAlarm::AlarmMap> newState);
 
+    // Whenever there is change on Supported attribute we should change Mask and State accordingly.
```

Comment:
The comment does not seem to explain the code but rather seems to contain an implementation detail of setsupportedvalue.

Maybe this should be rephrased saying `A change in supported value will result in a corresponding change in mask. For example: ....`

The comments in the header should help users of the API understand the API usage and requirements.

Reply:
Done